backend/sources/chat-plugin/src/main/java/co/airy/core/chat_plugin/config/AuthConfig.java

```diff
@@ -35,11 +36,15 @@ protected void configure(final HttpSecurity http) throws Exception {
                 .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
                 .addFilter(new JwtAuthenticationFilter(authenticationManager(), jwt))
                 .authorizeRequests()
-                .antMatchers("/actuator/**", "/ws.chatplugin").permitAll()
-                .mvcMatchers("/chatplugin.authenticate").permitAll()
                 .anyRequest().authenticated();
     }
 
+    @Override
+    public void configure(WebSecurity web) {
+        web.ignoring()
```

Comment:
So TIL that this is how you really open up routes. `permitAll` still runs the authentication middleware but it allows any principal to pass.